lib/index.js

```diff
@@ -126,11 +126,10 @@ function* createEverything() {
 
   var adminTestDatabaseClient = pgp(adminTestDatabase);
   yield adminTestDatabaseClient.task(alterPrivileges);
-
-  pgp.end();
```

Comment:
I suppose that if you are trying to move the database there, then perhaps shutting down the entire connection pool is inevitable ;)